moped-editor/src/views/projects/projectsListView/ProjectsListViewTable.js

```diff
@@ -298,8 +298,50 @@ const ProjectsListViewTable = ({ title, query, searchTerm, referenceData }) => {
       field: "updated_at",
       render: entry => new Date(entry.updated_at).toLocaleDateString("en-US"),
     },
+    {
+      title: "Task order",
+      field: "task_order",
+      hidden: true,
+      emptyValue: "-",
+      render: entry => {
+        // Empty value won't work in some cases where task_order is an empty array.
+        if (entry?.task_order.length < 1) {
+          return "-";
+        }
+        // Render values as a comma seperated string
+        let content = entry.task_order
+          .map(taskOrder => {
+            return taskOrder.display_name;
+          })
+          .join(", ");
+
+        return <div style={{ maxWidth: "265px" }}>{content}</div>;
+      },
+    },
+    {
+      title: "Contractor/Contract",
+      field: "contractor",
+      hidden: true,
+      emptyValue: "-",
+      render: entry => (entry.contractor === "" ? "-" : entry.contractor),
+    }, 
+    {
+      title: "Project DO#",
+      field: "purchase_order_number",
+      hidden: true,
+      emptyValue: "-",
+      render: entry =>
+        entry.purchase_order_number.trim().length === 0
+          ? "-"
+          : entry.purchase_order_number,
+    },
   ];
 
+  if (data) {
+    console.log(data[query.table])
+    console.log(query.query)
```

Comment:
Do you want to make these `.info()`s or take them out?

Reply:
I want to take them out! thank you for flagging them